src/pages/Admin/Admin.tsx

```diff
@@ -295,7 +323,10 @@ const VenueInfoComponent: React.FC<VenueDetailsPartProps> = ({
             )}
             <button
               className="btn btn-primary"
-              onClick={() => setShowCreateEventModal(true)}
+              onClick={() => {
+                history.push(`${match.url}/events`);
+                setShowCreateEventModal(true);
```

Comment:
i would have done it using a url parameter like `${match.url}/events?showModal=true`, but if this works then fine

Reply:
ah this would've been so much easier, didn't know you could do that